app/views/docs/functions.phtml

```diff
@@ -19,36 +19,36 @@ $runtimes = $this->getParam('runtimes', []);
 
 <p>You can deploy your code using the <a href="/docs/command-line">Appwrite CLI</a> tool, the Appwrite <a href="/docs/getting-started-for-server">Server API</a> or manually from the Appwrite console.</p>
 
-<h3><a href="/docs/functions#createTag" id="createTag">Create Tag</a></h3>
+<h3><a href="/docs/functions#createDeployment" id="createDeployment">Create Deployment</a></h3>
 
 <ul class="phases clear" data-ui-phases>
     <li>
         <h3>Unix</h3>
 
         <div class="ide margin-bottom" data-lang="bash" data-lang-label="Bash">
-            <pre class="line-numbers"><code class="prism language-bash" data-prism>appwrite functions createTag \
+            <pre class="line-numbers"><code class="prism language-bash" data-prism>appwrite functions createDeployment \
     --functionId=6012cc93d5a7b \
-    --command="node index.js" \
+    --entrypoint="index.js" \
     --code="/myrepo/myfunction"</code></pre>
         </div>
     </li>
     <li>
         <h3>CMD</h3>
 
         <div class="ide margin-bottom" data-lang="bash" data-lang-label="CMD">
-            <pre class="line-numbers"><code class="prism language-bash" data-prism>appwrite functions createTag ^
+            <pre class="line-numbers"><code class="prism language-bash" data-prism>appwrite functions createDeployment ^
```

Comment:
Can we also add autoDeployment functionality here? It's better to add it and let the developer remove it, than not talking about a feature. Probably applies to all CLI code examples.